09-Customizing-Lock/app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DashboardController < ApplicationController
+ include Secured
```

Comment:
@falvarisqueta: It is. When you browse the dashboard it will execute the before action in the `secured` controller concern.